style.css

```diff
@@ -0,0 +1,990 @@
+
+@media (max-width: 1265px) {
+  .text__wrap_character_down { margin-bottom: 88px; }
+
+  .score { margin-bottom: 65px; }
+
+  .score__item_icon {
+    width: 50px;
+    padding-bottom: 0;
+  }
+}
+
+@media (max-width: 1220px) {
+  .star { width: 92%; }
+
+  .star__item {
+    width: 20px;
+    margin-right: 0;
+  }
+
+  .section-desc__wrap-down { background-size: contain; }
+
+  .safe-body__item_corps { right: 70px; }
+}
+
+@media (max-width: 1150px) {
+  .text__wrap_character-down {
+    justify-content: flex-start;
+    margin-bottom: 40px;
+  }
+
+  .text__wrap_character-down,
+  .text__wrap_down { padding-left: 28px; }
+
+  .text__wrap_subtitle { margin-bottom: 15px; }
+
+  .text__wrap_section { height: 140px; }
+
+  .star,
+  .button_down { width: auto; }
+
+  .star__title { padding-left: 0; }
+
+  .score {
+    padding-left: 33px;
+  }
+
+  .score__item_icon {
+    width: 44px;
+    margin-right: 9px;
+  }
+
+  .button__item_big-down { padding-left: 14px; }
+
+  .section-desc__wrap-down { background-position: 99% 23%; }
+
+  .safe-body__item_corps {
+    top: 200px;
+    right: 40px;
+  }
+}
+
+@media (max-width: 1032px) {
+  .text__content { width: 97%; }
+
+  .text__wrap_character-down,
+  .text__wrap_down { padding-left: 20px; }
+  .text__wrap_character-down { height: 205px; }
+
+  .score {
+    padding-left: 21px;
+  }
+}
+
+@media (max-width: 900px) {
+  .text__wrap_character-down,
+  .text__wrap_down { padding-left: 8px; }
+
+  .text__title_down { padding-left: 0; }
+
+  .score { padding-left: 15px; }
+
+  .button__item_big-down { padding-left: 11px; }
+
+  .safe-body__item_corps {
+    top: 260px;
+    right: 32px;
+  }
+
+  .section-desc__wrap-down { background-position: 99% 31%; }
+
+  .item__icon_corps { max-width: 107%; }
+}
+
+/* Планшет */
+@media (max-width: 768px) {
+  .section-desc__wrap-down {
+    height: 538px;
+    background-position: 53% 16%;
+    background-size: 76%;
+  }
+
+  .safe-body__item_corps {
+    top: 75px;
+    right: 18px;
+  }
+
+  .item__icon_corps { max-width: 98%; }
+
+  .text__wrap_title-down {
+    padding-top: 59px;
+    margin-bottom: 10px;
+  }
+
+  .text__wrap_down { padding-left: 33px; }
+
+  .text__content_down { height: 163px; }
+
+  .text__wrap_character { 
+    justify-content: space-between;
+    width: 38%;
+  }
+  .text__wrap_character-down {
+    height: 238px;
+    margin-bottom: 0;
+    padding-top: 59px;
+    padding-left: 39px;
+  }
+
+  .text__wrap_subtitle { width: 100%; }
+
+  .text__subtitle { padding-left: 0; }
+
+  .text__wrap_section {
+    width: 100%;
+    height: 174px;
+    margin: 0;
+  }
+
+  .star__item {
+    width: 100%;
+    margin-right: 4px;
+  }
+
+  .score { display: none; }
+
+  .button_down { width: 70%; }
+
+  .button__item_big-down {
+    width: 40%;
+    padding-left: 0;
+    padding-right: 15px;
+  }
+}
+
+@media (max-width: 650px) {
+  .text__wrap_down,
+  .text__wrap_character { padding-left: 20px; }
+  .text__wrap_character {
+    width: 44%; 
+  }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)